toggles/webapp/toggles.ts

```diff
@@ -20,10 +20,10 @@ export default {
       description: 'Use the staging catalogue API',
     },
     {
-      id: 'stacksRequestService',
-      title: 'Items status and requesting',
+      id: 'showPhysicalItems',
```

Comment:
Are you sure we don't want to be developing behind 1 mega toggle? Are we planning on releasing this on its own, or will it be released as a larger part?

Reply:
yep, we'll be releasing bits incrementally